docs/source/daml/reference/choices.rst

```diff
@@ -72,10 +72,59 @@ Controllers
 
   The conjunction of **all** the parties are required to authorize when this choice is exercised.
 
-.. _daml-ref-anytime:
+.. _daml-ref-consumability:
+
+Consumability
+=============
+
+If no qualifier is present, choices are *consuming*: the contract is archived before the evaluation of the choice body.
+
+Note that the contracts created in the choice body are divulged to the signatories and controllers of the contracts as well as to the observers of the contract on which the choice was exercised. This behavior differs from that of *preconsuming* and *postconsuming* choices.
```

Comment:
It's not only contracts created. Any contract with an action on them (including fetches and exercises) is divulged.

Reply:
Solved in https://github.com/digital-asset/daml/pull/4134/commits/11f52f472e3cb698771d149a1e0d4ac0a464f02a